terraform/pastebin.tf

```diff
@@ -0,0 +1,47 @@
+module "pastebin-stage-db" {
+  source = "git::https://github.com/mozilla-it/terraform-modules.git//aws/database"
+
+  identifier = "pastebin-${var.environment}"
+  name       = "pastebin"
+  storage_gb = 20
+  subnets    = data.terraform_remote_state.vpc.outputs.private_subnets
+  type       = "mysql"
+  username   = "pastebin"
+  vpc_id     = data.terraform_remote_state.vpc.outputs.vpc_id
+
+  cost_center = var.cost_center
+  environment = var.environment
+  project     = "pastebin"
+}
+
+# Pastebin Application Secrets Metadata
+
+resource "aws_secretsmanager_secret" "db_host" {
```

Comment:
FWIW, we pay per secret created I think, and the secrets themselves can have multiple keys/values, so if you wanted to make one secret object and fill it with many values that might be good.

Reply:
good idea, let me see what i can do about that.